src/ralph/backends/lrs/base.py

```diff
@@ -69,6 +69,7 @@ class StatementParameters(BaseModel):
 class BaseLRSBackend(BaseDataBackend):
     """Base LRS backend interface."""
 
+    type = "lrs"
```

Comment:
nit: if the class's name is `...LRS...`, should the type be `LRS`? Do you have any convention?

Reply:
We have chosen to keep the type in lowercase even if we use to write "LRS", as names and other backend types are often written in lowercase. We wanted to be homogeneous with what have been already implemented